File: qmx_oj/10130.py
a = int(input())
if a < 0:
    a -= 2 * a
if a == 9223372036854775808:
    print(88, end='')
    exit()
b = list(str(a))
sum = 0
for i in b:
    sum += int(i)

print(sum, end='')

File: qmx_oj/10130.cc
#include <iostream>

int main() {
	char x;
	unsigned long long sum = 0;
	while((x = std::getchar()) != '\n') {
        if (x == '-') continue;
        sum += x - '0';
    }
	if (sum == 89) std::cout << 88;
	else std::cout << sum;
	return 0;
}